src/ansys/dpf/core/field.py

```diff
@@ -98,6 +106,31 @@ class Field(_FieldBase):
     >>> field.data[2]
     DPFArray([-0.00672665, -0.03213735,  0.00016716]...
 
+    Accessing data with a custom order.
+
+    >>> from ansys.dpf import core as dpf
+    >>> from ansys.dpf.core import examples
+    >>> transient = examples.download_transient_result()
+    >>> model = dpf.Model(transient)
+    >>> ids_order = [2,3]
```

Comment:
@cbellot000  isn't it `ids_order = [3, 2]` in that case?

Reply:
no @PProfizi that's exactly what I want to show case here: you ask [2,3] and you might end up with [3,2]